docs/maintaining.rst

```diff
@@ -196,13 +196,9 @@ Uploading to PyPI
 Once you've tagged the latest version of marbles, pull from GitHub to
 make sure your clone is up to date and clean, build both ``sdist`` and
 ``wheel`` packages for all three packages, and upload them with
-`twine`_::
+`twine`_. We have a `tox`_ rule to automate building and uploading::
```

Comment:
If we just run `tox` by itself, will it try to do the pypi upload too?

Reply:
Nope, it's not listed in `[tox]envlist`, which is the default set of envs to build with.

Reply:
Great, ty!